nca/NetworkConfig/NetworkConfigQuery.py

```diff
@@ -714,8 +714,8 @@ def __init__(self, config1, config2, output_config_obj=None):
     def get_query_type():
         return QueryType.ComparisonToBaseConfigQuery
 
-    def is_identical_topologies(self, check_same_policies=False):
-        if self.config1.peer_container != self.config2.peer_container:
+    def is_identical_configs(self, check_same_policies=False):
```

Comment:
I would keep the old name `is_identical_topologies`, as the term 'config' also refers to policies

Reply:
Ok, although I changed the name because in its second condition we check for eq. policies (although we get there only with the true flag)